linux_os/guide/system/software/integrity/crypto/configure_openssl_crypto_policy/rule.yml

```diff
@@ -22,6 +22,9 @@ severity: medium
 identifiers:
     cce@rhel8: 80938-4
 
+references:
+    nist: SC-12(3)
```

Comment:
Because OpenSSL supports asymmetric and symmetric encryption, SC-12(2) and SC-12(3) may be valid mappings.